app/views/shared/_sp_alert.html.erb

```diff
@@ -1,6 +1,6 @@
 <% alert = decorated_session.sp_alert(section) %>
 <% if alert %>
-  <%= render AlertComponent.new(text_tag: 'div') do %>
+  <%= render AlertComponent.new(text_tag: 'div', class: 'margin-bottom-4') do %>
     <%= raw sanitize(alert, tags: %w[a b strong em br p ol ul li], attributes: %w[href target]) %>
```

Comment:
random thought -- do we need to make sure to apply `usa-link` to the `<a>` tags in here?

Reply:
Good question. I thought we might, but it turns out that USWDS styles anchors in alerts automatically:

https://github.com/uswds/uswds/blob/v2.13.3/src/stylesheets/components/_alerts.scss#L111-L113

But! It looks like those styles changed in USWDS 3.x to only apply to `.usa-link` classes:

https://github.com/uswds/uswds/blob/v3.4.1/packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss#L96-L103

But! We configure global link styling, so the expected colors would be applied anyways:

https://github.com/18F/identity-idp/blob/7681e815696d2863028239266914c598ad22278f/app/assets/stylesheets/variables/_vendor.scss#L5